assets/app/views/deployments.html

```diff
@@ -68,18 +68,18 @@
               </dl>
 
               <dl class="dl-horizontal left indent">
-                <div ng-repeat="trigger in deploymentConfig.triggers">
+                <div ng-repeat="trigger in deploymentConfig.spec.triggers">
                   <span ng-switch="trigger.type">
                     <span ng-switch-default>{{trigger.type}}</span>
                     <span ng-switch-when="ImageChange">
-                      <div ng-switch="deploymentConfig.template.strategy.type">
+                      <div ng-switch="deploymentConfig.spec.strategy.type">
                         <div ng-switch-when="Recreate">
```

Comment:
no idea why we're checking the deployment strategy type here. @ironcladlou, do ImageChange triggers only apply to `Recreate` strategy deployments?